asab/api/service.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, app, service_name="asab.ApiService"):
 				with open(path) as f:
 					self.Manifest = json.load(f)
 			except Exception as e:
-				L.exception(e)
+				L.log(e)
```

Comment:
This is not correct.
`L.exception("Error when reading manifest")` ... is correct.